apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -47,7 +47,7 @@ patterns() ->
 -spec forbidden( Mod :: module(), OperationID :: atom() ) -> boolean().
 forbidden(Mod, OpId) ->
     OpSpec = Mod:operation(OpId),
-    [ #{ tags := Tags } | _ ] = maps:values(OpSpec),
```

Comment:
Looks like you can't set debug on individual methods. Unless each OpId will only contain a single method - so you would need a standalone DeleteTx OpId with only the DELETE method. In which case this is guaranteed to be a single list entry so could safely match on a single entry.

Reply:
Basically the combination of URI + method defines a single operation that comes with a unique OperationID. The old structure used to be
```
#{ OpID1 => #{method1 => OpSpec},
   OpID2 => #{method2 => OpSpec2},
   ....
   }```
Since there can be only one method per OperationID, the key-value relation is a bit unnecessary and could be misleading (implying that there could be another method there) so I simplified it, moving the method to the OpSpec instead.

Reply:
So this could safely be `[ #{ tags := Tags } ] = maps:values(OpSpec)`

To save future readers hunting around for the case there might be multiple methods

Reply:
Well not really: the old code was a bit misleading.
The structure used to be:
```
operations() ->
    #{'GetChannelByPubkey' =>
      #{get =>
            #{parameters =>
                  [[{"in","query"}, 
                    {"name","int-as-string"},
                    {"description",
                     "If this flag is set to true, the response will have all integers set as strings"},
                    {"required",false},
                    {"schema",[{"type","boolean"},{"default",false}]}],
                   [{"in","path"},
                    {"name","pubkey"},
                    {"description","The pubkey of the channel"},
                    {"required",true},
                    {"type","string"},
                    {"example",
                     "ch_2tceSwiqxgBcPirX3VYgW3sXgQdJeHjrNWHhLWyfZL7pT4gZF4"}]],
              path => <<"/v3/channels/{pubkey}">>,
              responses =>   
                  #{200 => #{<<"$ref">> => <<"/components/schemas/Channel">>},
                    400 => #{<<"$ref">> => <<"/components/schemas/Error">>},
                    404 => #{<<"$ref">> => <<"/components/schemas/Error">>}},
              tags => [<<"external">>,<<"channel">>]}},

....
```
where `method1` and `method2` were placeholders for `get` or `post`. Calling ` Mod:operation/1` with a Operation ID (for example with `GetChannelByPubkey`) would return the whole nested map with one key (being `get` from the example above). This was named as `OpSpec` above and thus the call to `maps:values/1` to retrieve the actual OpSpec :) Now the same structure would look like this:

```
```
operations() ->
    #{'GetChannelByPubkey' =>
            #{parameters =>
                  [[{"in","query"}, 
                    {"name","int-as-string"},
                    {"description",
                     "If this flag is set to true, the response will have all integers set as strings"},
                    {"required",false},
                    {"schema",[{"type","boolean"},{"default",false}]}],
                   [{"in","path"},
                    {"name","pubkey"},
                    {"description","The pubkey of the channel"},
                    {"required",true},
                    {"type","string"},
                    {"example",
                     "ch_2tceSwiqxgBcPirX3VYgW3sXgQdJeHjrNWHhLWyfZL7pT4gZF4"}]],
              path => <<"/v3/channels/{pubkey}">>,
              responses =>   
                  #{200 => #{<<"$ref">> => <<"/components/schemas/Channel">>},
                    400 => #{<<"$ref">> => <<"/components/schemas/Error">>},
                    404 => #{<<"$ref">> => <<"/components/schemas/Error">>}},
              tags => [<<"external">>,<<"channel">>],
              method => "get"},

....
```
...and the ` Mod:operation/1` would return the actual OpSpec :) Since the method is only one, it is moved to the OpSpec itself.

Calling `maps:values/1` on the OpSpec would make little sense...

Reply:
My bad - looking at it backwards